@navikt/core/tokens/tailwind/src/tests/reducer.test.ts

```diff
@@ -0,0 +1,42 @@
+import Reducer from "../reducer";
+
+describe("Check correct parsing of tailind tokens", () => {
+  test("Check reducer function", () => {
+    expect(
+      Reducer(
+        {
+          "shadow-focus-inverted":
+            "0 0 0 3px var(navds-semantic-color-focus-inverted)",
+          "z-index-focus": 10,
+          "global-color-blue-300": "rgba(102, 165, 244, 1)",
+          "semantic-color-interaction-primary":
+            "var(--navds-gloab-color-blue-300)",
```

Comment:
Jeg tenker vi burde bestemme oss for å enten bruke CSS custom properties (`var(--navds-gloab-color-blue-300)`) eller å bruke verdiene direkte (`rgba(102, 165, 244, 1)`). Å ha en blanding som her virker rart for meg.

Reply:
Ah ja siden dette vil være en helt standalone pakke så er det ingen garanti at css-variabler blir satt i browser. Endrer slik at fargene blir brukt direkte + forenkler scriptet da.